data/systemd/insights-client-boot.service

```diff
@@ -0,0 +1,30 @@
+# This file is part of insights-client.
+#
+# Any changes made to this file will be overwritten during a software update. To
+# override a parameter in this file, create a drop-in file, typically located at
+# /etc/systemd/system/insights-client-boot.service.d/override.conf Put the desired
+# overrides in that file and reload systemd.
+#
+# For more information about systemd drop-in files, see systemd.unit(5).
+
+[Unit]
+Description=Run Insights Client at boot
+Documentation=man:insights-client(8)
+After=network.target
+ConditionPathExists=/etc/insights-client/.run_insights_client_next_boot
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/insights-client --retry 3
+Restart=no
+WatchdogSec=900
+CPUQuota=30%
+MemoryLimit=2G
```

Comment:
`MemoryMax=` replaces `MemoryLimit=`, no need for both.

Reply:
This also needs to work in RHEL 7 -- I'm not sure whether you can just use `MemoryLimit` as an alias or whether you can define both, but we should go for max compatibility here.

Reply:
Gotcha. :+1: